.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,10 +6,6 @@
 
 <!-- What have you changed? Consider adding a screenshot or GIF. -->
 
-## QA notes
-
-<!-- How has the expected behaviour changed? What testing strategies have you used? -->
-
```

Comment:
I thought we were going to keep this section in Kedro-Viz (because it's front-end), but not use it in Kedro? It's gonna be one of the few differences between the repos.

Reply:
Oh, okay. I misunderstood the template would be identical.. Updated again